tests/test-data.ldif

```diff
@@ -127,4 +127,86 @@ objectclass: iNetOrgPerson
 sn: User
 cn: Board3
 mail: success+boardthree@simulator.amazonses.com
-userPassword: Password1
```

Comment:
Everything below is an unchanged copy/paste from `tests/add-groups.ldif` it's just an amalgamation of the two files.